index.html

```diff
@@ -883,6 +941,18 @@ <h2>Determine if a performance entry buffer is full</h2>
         <li>Return true.</li>
       </ol>
     </section>
+    <section data-link-for="PerformanceEntry">
+      <h2>Generate a navigationId</h2>
+      <p>When asked to <dfn class="export">generate a navigationId</dfn> for a
+      <a>PerformanceEntry</a> <var>entry</a>, run the following steps:</p>
+      <ol>
+        <li>Let <var>type</var> be <var>entry</var>'s <a>entryType</a>.</li>
+        <li>Let <var>startTime</var> be <var>entry</var>'s <a>startTime</a>.
+        </li>
+        <li>Return the [=string/concatenation=] of « <var>type</var>,
+        <var>time</var> » using U+002D HYPHEN-MINUS.</li>
```

Comment:
concatenation is defined for a list of strings, so I think you need to convert the timestamp to a string first.